src/app/home/home.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="d-flex justify-content-center mt-5" *ngIf="isLoading && !error">
+    <div class="spinner-border" role="status">
+        <span class="sr-only">Loading...</span>
+    </div>
+</div>
+
+<div class="alert alert-danger mt-3" *ngIf="error">
+    <h1>An error occurred!</h1>
+    <p>{{ error }}</p>
+</div>
+
+<div class="container" *ngIf="!isLoading && !error">
+
+    <app-info-card class="row text-center mt-4" [data]="data"></app-info-card>
+
+    <div class="row">
+        <app-charts class="col-lg-6" [chartData]="chartData"
+            [labels]="['TotalConfirmed','TotalRecovered','TotalDeaths']" [numberOfData]="20">
```

Comment:
[labels] should probably be declared in a component (not a huge issue, just a better practice)